### DISCUSSION TITLE
Fonts reload

### DISCUSSION THREAD

**Discussion by Vovcharaa** [gotenberg/gotenberg#998] [Q&A]
Is it possible to reload fonts when content of mounted volume with fonts is changed?
I have mounted S3 bucket with fonts using `mountpoint-s3-csi-driver` but after adding new fonts I need to restart Pod to use them.

**Comment by gulien** [comment]
Hello @Vovcharaa,

I’m not familiar with the subject, but, according to [this article](https://davelage.com/posts/rebuild-font-cache-linux/), you may refresh the font cache on Linux. Still, it may also require a program restart so it seems that restarting the Pod is the way to go 🤔

**Reply by Vovcharaa** [reply]
I tried cache rebuilding but there is no way to force restart chrome/libreoffice because they don't see new fonts without restart.
